bin/awslocal.bat

```diff
@@ -1 +1,3 @@
+@echo off
```

Comment:
Thanks @4lex1v . I'm not too familiar with bat scripts in Windows. Can you please explain a bit how this works? My understanding is that this command simply disables log output to stdout, so it is not clear to me how this can solve the `FileNotFoundError` in #15..?

Reply:
Hi @whummer. I'm not an expert in batch scripting either, so my understanding could be imprecise. `@echo off` doesn't disable log output to stdout, but disables command echoing, i.e Windows won't show you the batch command it's about to execute, only the output of that command, it's like logging the command that it's about to execute. 

I haven't had an issue with STS, like the issue mentions, but command echoing breaks pipelines on Windows, cause you get the command as well as the output, e.g `awslocal cloudwatch list-metrics | jq` would fail since the output has the echo beside the json response. 

I'd assume that something similar may happen in #15.

Reply:
Ok, thanks for clarifying. Happy to merge this one (although I think #15 may require a different solution). We'll release a new version shortly. Thanks